### ISSUE TITLE
Add hedera-json-rpc-relay service

### ISSUE THREAD

**Issue by Nana-EC** [hashgraph/hedera-local-node#88]
### Problem

The local-node currently doesn't have a relay service since it was waiting on a stable version.
A v0.1.0 now exists and should be added

### Solution

- Add the `relay` service to  the docker compose using the following template
```yaml
relay:
    container_name:  json-rpc-relay
    image: "ghcr.io/hashgraph/hedera-json-rpc-relay:${RELAY_IMAGE_TAG}"
    restart: "unless-stopped"
    ports:
      - 7546:7546
    pull_policy: always    
    restart: unless-stopped
    tty: true
```
- Add mem service details. may be similar to web3 service 
- Add `depends_on` rest and `network-node`
- Add the `RELAY_IMAGE_TAG` env variable to modify the relay version. Start with 0.1.0
- Add env_variable to service details matching relay [local configuration](https://github.com/hashgraph/hedera-json-rpc-relay#configuration). Omit the `LOCAL_NODE` variable noted

**Comment by nathanklick** [comment]
Duplicate of #41 
All work will be done under #41
